ext/calendar/calendar.c

```diff
@@ -173,13 +173,14 @@ static void _php_cal_info(int cal, zval *ret)
    Returns information about a particular calendar */
 PHP_FUNCTION(cal_info)
 {
-	zend_long cal = -1;
+	zend_long cal;
+	zend_bool cal_is_null = 1;
 
-	if (zend_parse_parameters(ZEND_NUM_ARGS(), "|l", &cal) == FAILURE) {
+	if (zend_parse_parameters(ZEND_NUM_ARGS(), "|l!", &cal, &cal_is_null) == FAILURE) {
```

Comment:
Hm, it looks like for this function we already document `-1` as the default. Maybe we just want to specify that in the stub?

Reply:
Oh, that's a bit sad, but makes sense.

Reply:
I think things got mixed up, you've now updated unixtojd() to use -1 and left the null hear. I meant that the other way around :)

Reply:
Uh, sorry. I think I'm just tired and messed up the search. I should stop for today. :)

Reply:
Hopefully the amended fixup commit fixed the issue